src/App.js

```diff
@@ -1,29 +1,32 @@
 import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
-import Map from './components/Map';
+import SearchResults from './context/SearchResults';
 import List from './components/List';
 import Settings from './components/Settings';
 import Help from './components/Help';
 import Nav from './components/Nav';
+import Map from './components/Map';
 
 import './App.css';
 
 const App = () => {
   return (
-    <Router>
-      <div className="app">
-        <main>
-          <Switch>
-            <Route path="/" exact component={Map} />
-            <Route path="/list" component={List} />
-            <Route path="/settings" component={Settings} />
-            <Route path="/help" component={Help} />
-          </Switch>
-        </main>
-        <Nav />
-      </div>
-    </Router>
+    <SearchResults>
+      <Router>
+        <div className="app">
```

Comment:
Could we simplify this by removing the `<div>` and putting the class name on `<main>`?

Reply:
I don't think so -- especially with that nav bar bug we have,  the outer div needs to stay